app/services/analytics.rb

```diff
@@ -6,18 +6,21 @@ def initialize(user, request)
 
   def track_event(event, attributes = {})
     analytics_hash = {
-      event: event,
-      properties: attributes.except(:user_id),
+      event_properties: attributes.except(:user_id),
       user_id: attributes[:user_id] || uuid
     }
 
-    ANALYTICS_LOGGER.info(analytics_hash.merge!(request_attributes))
+    ahoy.track(event, analytics_hash.merge!(request_attributes))
   end
 
   private
 
   attr_reader :user, :request
 
+  def ahoy
+    @ahoy ||= Rails.env.test? ? FakeAhoyTracker.new : Ahoy::Tracker.new(request: request)
+  end
```

Comment:
Yeah that seems fair enough, would we have to implement all the methods or just a no-op `initialize`?